### PR TITLE
Updated installation instructions on FreeBSD:

### PR DESCRIPTION
**Pull request by matsduf**
* Corrected path for included configuration files.
* Added needed enabling of modules in Apache.
* Removed uneeded use of wget for fetch instead (and removed
  installation of wget)
* Editorial updates

The current instructions are incomplete or wrong.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -109,50 +113,47 @@ sudo systemctl reload apache2
...
-```
+* Start Apache:
+ 
+``service apache24 start``
 
 * Install Zonemaster Web GUI
 ```sh
-wget https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip -O temp.zip
-mkdir -p  /var/www/html/zonemaster-web-gui
+fetch https://github.com/zonemaster/zonemaster-gui/releases/download/v3.0.1/zonemaster_web_gui.zip
```
Comment (sandoche2k) [inline docs/Installation.md]: Should be 3.0.2

**Comment by sandoche2k** [comment]
@pamasse since the file name now is not "temp.zip", you need to change for all distributions as @matsduf has done

File: docs/Installation.md (hunk 1 above)
Comment (matsduf) [reply docs/Installation.md]: That will be fixed at release time. We should probably have 3.0.3 since we have added things after.

**Comment by matsduf** [comment]
@sandoche2k, that is done in #53.

**Comment by matsduf** [comment]
Rebased after #53 was merged to develop branch.

**Pull request merged**